proto/buf/alpha/registry/v1alpha1/scim_token.proto

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "google/protobuf/timestamp.proto";
+
+message SCIMToken {
+  string id = 1;
+  google.protobuf.Timestamp create_time = 2;
+  google.protobuf.Timestamp expire_time = 3;
+}
+
+// TokenSCIMService is the SCIM Token service.
+service SCIMTokenService {
+  // CreateToken creates a new token suitable for authentication to the SCIM API.
```

Comment:
```suggestion
  // CreateToken creates a new token suitable for authentication to the SCIM API.
  //
  // This method requires authentication.
```
nit: assuming this requires authentication as well, but looks like it's copied over from the existing token.proto.